code/game/objects/items/weapons/tools/mods/mod_types.dm

```diff
@@ -494,6 +497,7 @@
 	icon_state = "expand"
 	desc = "A bulky adapter which more modifications to be attached to the tool. A bit fragile but you can compensate. Due to its complex design it takes an extraordinarily long time to remove with tools."
```

Comment:
```suggestion
	desc = "A bulky adapter which more modifications to be attached to the tool. A bit fragile but you can compensate. Due to its complex design it can not be remove form the tool."
```